src/VisualStudio/Core/Def/Implementation/ProjectSystem/RuleSets/VisualStudioRuleSetManager.RuleSetFile.cs

```diff
@@ -150,6 +152,12 @@ public void Dispose()
                 _disposalCancellationSource.Dispose();
             }
 
+            public ValueTask DisposeAsync()
+            {
+                Dispose();
+                return ValueTaskFactory.CompletedTask;
+            }
```

Comment:
❔ Why are we making this one implement `IAsyncDisposable`?

Reply:
i'll undo these.